app/controllers/main_window_controller.py

```diff
@@ -1,7 +1,6 @@
-from loguru import logger
-
 from PySide6.QtCore import QObject, Slot
 from PySide6.QtWidgets import QPushButton
+from loguru import logger
```

Comment:
I agree that the whitespace should be removed, but import order groups should be sorted alphabetically.

> Imports should be grouped in the following order:
> 
>     Standard library imports.
>     Related third party imports.
>     Local application/library specific imports.

Your grouping choice is correct, but loguru should be imported before PySide6 imports (L comes before P)

Reply:
I just Used Optimize Imports feature in PyCharm and it arranged it